src/core/dxf/qgsdxfexport.h

```diff
@@ -267,6 +267,24 @@ class CORE_EXPORT QgsDxfExport : public QgsLabelSink
      */
     QgsRectangle extent() const { return mExtent; }
 
+    /**
+     * Use only selected features
```

Comment:
This could be added to QgsDxfExport::Flag instead, and avoid the need for the new getter/setter

Reply:
Done. Much better, thanks for the suggestion.